shor/quantum.py

```diff
@@ -53,6 +53,19 @@ def measure_bits(self):
 
         return measure_bits
 
+    def to_registers(self):
+        registers = []
+        for m in filter(lambda l: type(l) == Qbits, self.layers):
+            for q in m._qbits:
```

Comment:
qbit_registers.append(m)

Reply:
don't we want to send the qbits over not the class that contains them?

Reply:
nvm I think I see what your getting at